tests/qm-sanity-test/main.fmf

```diff
@@ -0,0 +1,6 @@
+summary: qm sanity test
```

Comment:
missing polarion-id

Reply:
I split the sanity test into 6 tests. Once confirm that each test description is OK, I will import them into Polarion to generate Polarion id.

Reply:
added polarion-id